data/hai missions.txt

```diff
@@ -1061,3 +1061,254 @@ mission "Hai Honeymoon"
+			`	While you tell her stories of many different planets you've visited, she seems to be particularly interested in Earth. "I've seen a bit of human architecture in videos of planets like Allhome, but I can't even imagine what your capital must look like. Tens of thousands of years of another species' history, it sounds even more exciting than what I heard about our landmarks in Hai-home. Captain, do you think that someday, when we no longer need to be kept hidden from humanity, that I'll be able to visit Earth, and see the beauty of your capital?" Seeing how little she knows about humanity, it's not surprising that she is unaware of the state that Earth is in.`
+			choice
+				`	(Avoid talking about Earth.)`
+					goto avoid
+				`	(Lie about Earth.)`
+					goto fake
+				`	(Tell her about how Earth actually is.)`
+					goto truth
+			label avoid
+			`	You tell her that you haven't ever visited the human capital, and that you never had much interest in learning about it. "Oh, sorry about that, shouldn't have assumed you knew everything, you're a starship captain after all, not a historian, I'll just ask someone else when I get the chance.", says Nanachi.`
+					goto board
+			label fake
+			`	You start making up stories about how beautiful the skies of Earth are. You describe forests teeming with life, and do your best to remember ancient civilizations and their monuments. From the looks of it she believes you, and your lie seems to have sated her curiosity for now. If the day where the Hai are allowed to move around human space ever comes, though, she will no doubt be extremely disappointed.`
+					goto board
+			label truth
+			`	She looks at you in confusion, as if trying to understand what you mean, so you explain it to her. "Centuries of widespread deforestation and pollution badly damaged the ecosystem. If you ever managed to visit, you'd be better off going to museums or monuments, but even then, there are many thieves roaming the streets, and it is very dangerous to be out at night. Lots of folks born there just think about how to get out."`
+			`	You keep telling her about the issues facing humanity's birthplace, and her previous eagerness to learn about the planet fades. "How could your ancestors let their own home become like this?"`
+			choice
+				`	"It's rather complicated, and I'm not a historian, sociologist, or economist, but I can try and explain what I do know."`
+			`	You recite what you read in school books or saw in documentaries, telling her about how many humans since millennia ago refused to take care of the planet, and how that piled up over hundreds of years.`
```

Comment:
Always be careful referencing current (our) events, statements you make can become untrue or you may cause some groups to take issue. While this will probably be fine do make sure you have made a conscious decision to include it.

Reply:
Well, even if the current events (Climate change) are reversed, there's still a ton of other issues regarding the planet's usage that will still exist, or soon exist.  It's mainly to contrast how Hai-Home is well taken care of.

Reply:
Instead of talking about the state of the world now (as in 21st century now) and as some claim it'll be going forward, this seems like a great opportunity to shed some light on the in-game state of Earth in terms of the overpopulation, the crime, the general state of the cities, etc. stuff that's actually mentioned in game already (planet description, Shuttle intro with James, the human space wiki page) but that we could expand upon.
It does seem though that we could cut back on some of this bit of the mission; the other two choices only lead to single paragraphs, but this choice is leading to 7 paragraphs/lines and 2 choices.

Reply:
This also seems like a great place to make comparisons to the Unfettereds' mistreatment of their own worlds.

Reply:
Nanachi most likely doesn't know much, if anything beyond what the landing descriptions give; I would expand upon that mission/guy who asks for the "whole of human history" to be a far better person to go into detail.  (Would be an interesting chain of missions where you collect some scientists from all the races, do some sort of "alien meet and greet" for everyone...)

Reply:
Nanachi isn't likely to know much about the Unfettered...  I mean, she's a brand new captain, already having issues finding jobs...  Maybe as a post-"campaign" mission chain where you take her to Earth after she's been a captain for a while, or if you talk to that guy who wanted the "history of the Humans" data.